resources/test-data.md

```diff
@@ -17,13 +28,13 @@ To test a checked-in user in the Demoshop, please use the following test data:
 
 ### Checkout test data for Norway
 
-{:.table .table-striped}
-| Type            | Data                         | Description                                                           |
-| :-------------- | :--------------------------- | :-------------------------------------------------------------------- |
-| `Email`         | `olivia.nyhuus@payex.com`    | The e-mail address of the payer.                                      |
-| `Mobile number` | `+47 98765432`               | The mobile phone number of the payer. Format Norway: `+47 99999999`.  |
-| `SSN`           | `{{ page.consumer_ssn_no }}` | The social security number of the payer. Format Norway: `DDMMYYXXXXX` |
-| `ZipCode`       | `1642`                       | The city zip code. Format Norway: `XXXX`                              |
+{:.table .table-striped }
+| Type              | Data                         | Description                                                           |
+| :--------------   | :--------------------------- | :-------------------------------------------------------------------- |
+| `{{"Email getClass"}}` | `olivia.nyhuus@payex.com`    | The e-mail address of the payer.                                      |
```

Comment:
What is `{{"Email getClass"}}` doing?

Reply:
ops, that was just something a tried before..! adding the | trim. it's supposed to look like the others